--- a/src/main/java/com/elfmcys/yesstevemodel/resource/YSMBinaryDeserializer.java
+++ b/src/main/java/com/elfmcys/yesstevemodel/resource/YSMBinaryDeserializer.java
@@ -1,5 +1,6 @@
 package com.elfmcys.yesstevemodel.resource;
 
+import com.elfmcys.yesstevemodel.YesSteveModel;
 import com.elfmcys.yesstevemodel.resource.pojo.RawYsmModel;
 import rip.ysm.security.YSMByteBuf;
 import io.netty.buffer.Unpooled;
@@ -77,8 +78,7 @@
             }
 
         } catch (Throwable t) {
-            System.out.println("ERROR");
-            t.printStackTrace(System.out);
+            YesSteveModel.LOGGER.warn("Failed to deserialize binary model", t);
         }
     }
 
