--- a/src/main/java/com/elfmcys/yesstevemodel/audio/AudioStreamWrapper.java
+++ b/src/main/java/com/elfmcys/yesstevemodel/audio/AudioStreamWrapper.java
@@ -1,5 +1,6 @@
 package com.elfmcys.yesstevemodel.audio;
 
+import com.elfmcys.yesstevemodel.YesSteveModel;
 import net.minecraft.client.sounds.AudioStream;
 import org.jetbrains.annotations.NotNull;
 import org.lwjgl.BufferUtils;
@@ -27,7 +28,7 @@
             this.currentStream = streamFactory2.openStream();
             this.audioFormat = this.currentStream.getFormat();
         } catch (IOException | UnsupportedAudioFileException e) {
-            e.printStackTrace();
+            YesSteveModel.LOGGER.warn("Failed to open audio stream", e);
             throw e;
         }
     }
@@ -52,7 +53,7 @@
                         return EMPTY_BUFFER;
                     }
                 } catch (Throwable th) {
-                    th.printStackTrace();
+                    YesSteveModel.LOGGER.warn("Failed to reset audio stream", th);
                     return EMPTY_BUFFER;
                 }
             }
