--- a/src/main/java/com/elfmcys/yesstevemodel/util/VarHandleHelper.java
+++ b/src/main/java/com/elfmcys/yesstevemodel/util/VarHandleHelper.java
@@ -1,5 +1,7 @@
 package com.elfmcys.yesstevemodel.util;
 
+import com.elfmcys.yesstevemodel.YesSteveModel;
+
 import java.lang.invoke.MethodHandles;
 import java.lang.invoke.VarHandle;
 import java.util.Optional;
@@ -9,7 +11,7 @@
         try {
             return Optional.of(MethodHandles.privateLookupIn(cls, MethodHandles.lookup()).findVarHandle(cls, str, cls2));
         } catch (IllegalAccessException | NoSuchFieldException e) {
-            e.printStackTrace();
+            YesSteveModel.LOGGER.warn("Failed to find var handle for field '{}' in {}", str, cls.getName(), e);
             return Optional.empty();
         }
     }
